mkosi.images/base/mkosi.conf.d/10-debian-ubuntu/mkosi.build.chroot

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+if [ ! -d deb/ ]; then
+    exit 0
+fi
+
+# We transplant the debian/ folder from the debian package sources into the upstream sources.
+mount --mkdir --bind "$SRCDIR"/deb/debian "$SRCDIR"/debian
+
+# We hide the patches/ directory by mounting an empty directory on top so they don't get applied.
+TMP=$(mktemp -d)
+mount --bind "$TMP" "$SRCDIR"/debian/patches
+
+# We can't specify the build directory so we use a bind mount to make sure the build artifacts get cached.
+mount --mkdir --bind "$BUILDDIR" "$SRCDIR"/obj-x86_64-linux-gnu
```

Comment:
architecture should be derived from meson or another source that actually matches what is being built